spec/features/medicaid_application_with_maximum_info_spec.rb

```diff
@@ -81,7 +81,7 @@
       expect(page).to have_content("Are you self-employed?")
       click_on "Yes"
 
-      expect(page).to have_content("Do you get income that's not from a job?")
+      expect(page).to have_content("Do you get income that’s not from a job?")
```

Comment:
Is this something we should keep an eye out for? Using the proper apostrophe instead of a single quote?

Reply:
Because, if so, I got this right here ready for us: https://github.com/codeforamerica/michigan-benefits/tree/jro-apostrophes

Reply:
I copy-pasted the title from Trello and it had the proper apostrophe so I figured I'd go with it -- 👏

Reply:
I love the consistency of that commit  / future PR!